react-form-controls/my-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled(props) {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const formData = new FormData(form);
+    const formValues = Object.fromEntries(formData.entries());
+    console.log('Controlled', formValues);
```

Comment:
Not necessary. Since this is a controlled component, we should just be logging `username` & `password` directly